packages/preferences/src/browser/style/navbar.css

```diff
@@ -0,0 +1,35 @@
+/********************************************************************************
+ * Copyright (C) 2020 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+.theia-settings-container .settings-navbar {
+  position: sticky;
+  float: left;
+  list-style: none;
+  top: 1em;
+  padding: 0;
+  margin: 0;
+  width: 150px;
```

Comment:
Do we need to hard code width, min-widget and other sizes? Some look like very big numbers in this PR I wonder how it should look like on the small screen. Can we get rid of it like it is done for other widgets?

Reply:
I have replaced the pixel values of several rules with em values to make them scale with font size. This looks okay at window widths as low as 600px. Is this okay?